pytest_splunk_addon_ui_smartx/components/dropdown.py

```diff
@@ -101,6 +101,34 @@ def select(self, value):
         else:
             raise ValueError("{} not found in select list".format(value))
 
+    def select_nested(self, values):
+        """
+        Selects the values we want from the type list in defined order
+            :param values: Dropdown values list in order we want to select
+            :return: Returns True if successful, otherwise raises an error
+        """
+        if not isinstance(values, list):
+            raise ValueError("{} has to be of type list".format(value))
+
+        self.add_input.click()
+        popoverid = "#" + self.add_input.get_attribute("data-test-popover-id")
+        dropdown_selector = (
+            ' [data-test="item"]:not([data-test-selected="true"]) [data-test="label"]'
```

Comment:
can't you just get text from `[data-test="item"]`?

<img width="1117" alt="image" src="https://github.com/splunk/addon-factory-smartx-ui-test-library/assets/142901247/dc7f0d07-c350-49c5-8759-a6baa00d48f2">

Reply:
I reused existing code part - probably `:not([data-test-selected="true"]` was added to not allow choosing already chose option but I don't see a reason for such behaviour. Updated code as per your suggestion for both `select()` and `select_nested` methods.

Reply:
if it is a menu, how it can be _selected_?